src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ProductVariant.xml

```diff
@@ -20,26 +20,36 @@
 
         <attribute name="normalization_context">
             <attribute name="groups">
-                <attribute>product_variant:read</attribute>
+                <attribute>admin:product_variant:read</attribute>
+                <attribute>shop:product_variant:read</attribute>
             </attribute>
         </attribute>
```

Comment:
Same question as in ProductTaxon. And by the way why in this file ItemOperations are above CollectionOperations? 😄

Reply:
good question

Reply:
IMO, the same as in ProductTaxon, there is no need for this config